packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -712,6 +712,33 @@ describe('Call Tests', () => {
       {file: 'call', method: 'updateMedia'}
     );
   });
+
+  describe('#addSessionConnection', () => {
+    let call;
+
+    beforeEach(async () => {
```

Comment:
any reason why this is async?

Reply:
It is the Github Copilot that must have auto-filled it. Sorry, I overlooked it. Removed it now